doc/content/lorawan/db-dbm-dbi-dbd/_index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "dB, dBm, dBi, and dBd"
+section: The Things Advanced
+description: ""
+weight:
+---
+
+In this chapter we learn about some units in brief that are used to measure the performance of transceivers (gateways and end devices) and antennas.
+
+## dB (decibel)
+
+Decibel expresses the ratio between two power levels, for example, the output power and the input power of an amplifier. The input power (P<sub>in</sub>) and output power (P<sub>out</sub>) are usually measured in Watt (W) or milliwatt (mW).
+
+For example, an amplifier with an output of 1000 Watt when the input is 1 Watt, its power ratio or amplification factor can be calculated as,
```

Comment:
self-review - rewording
For example, if an amplifier turns a 1 W signal into a 1000 W signal, its power ratio or amplification factor can be calculated as,